float32 degrees
---
bool status
